core/templates/rid_owner.h

```diff
@@ -350,24 +350,25 @@ class RID_Alloc : public RID_AllocBase {
 				if (validator & 0x80000000) {
 					continue; //uninitialized
 				}
-				if (validator != 0xFFFFFFFF) {
+				if (chunks != nullptr validator != 0xFFFFFFFF) {
```

Comment:
Please try to submit code, that you have actually verified, that it can compile. This doesn't.